fn main() {
    println!("Hello, world!");
    eprintln!("[ERROR] 予期しないエラーです！");
    // std::io::stdout().flush().unwrap();
}
